pkg/moov/transfer_models.go

```diff
@@ -371,6 +372,28 @@ type TransferOptions struct {
 	DestinationOptions []PaymentMethod `json:"destinationOptions,omitempty"`
 }
 
+// LineItems contains Items for nested structure
+type LineItems struct {
+	Items []LineItem `json:"items"`
+}
+
+// LineItem represents a single item in a transfer.
+type LineItem struct {
+	Name      string           `json:"name"`
+	ProductID *string          `json:"productId,omitempty"`
+	BasePrice Amount           `json:"basePrice"`
+	Quantity  int              `json:"quantity"`
+	Options   []LineItemOption `json:"options,omitempty"`
+}
+
+// LineItemOption represents an option for a line item.
+type LineItemOption struct {
+	Group         string `json:"group"`
+	Name          string `json:"name"`
+	PriceModifier Amount `json:"priceModifier"`
```

Comment:
`priceModifier` is also optional and an `AmountDecimal` type:
```suggestion
	PriceModifier *AmountDecimal `json:"priceModifier,omitempty"`
```